samples/bluetooth/mesh/light_lc/prj.conf

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+
+# General configuration
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_FCB=y
+CONFIG_SETTINGS=y
+CONFIG_HWINFO=y
+CONFIG_LOG=y
+CONFIG_LOG_PRINTK=y
+
+CONFIG_DK_LIBRARY=y
+CONFIG_BT_MESH_DK_PROV=y
+
+CONFIG_PWM=y
+
+# Bluetooth configuration
+CONFIG_BT=y
+CONFIG_BT_COMPANY_ID=0x0059
+CONFIG_BT_DEVICE_NAME="Mesh Light LC"
+CONFIG_BT_L2CAP_TX_MTU=69
+CONFIG_BT_L2CAP_TX_BUF_COUNT=8
+CONFIG_BT_LL_SW_SPLIT=y
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_SETTINGS=y
+
+# Disable unused Bluetooth features
+CONFIG_BT_CTLR_DUP_FILTER_LEN=0
+CONFIG_BT_CTLR_LE_ENC=n
+CONFIG_BT_DATA_LEN_UPDATE=n
+CONFIG_BT_PHY_UPDATE=n
+CONFIG_BT_CTLR_CHAN_SEL_2=n
+CONFIG_BT_CTLR_MIN_USED_CHAN=n
+CONFIG_BT_CTLR_PRIVACY=n
+
+# Bluetooth Mesh configuration
+CONFIG_BT_MESH=y
+CONFIG_BT_MESH_RELAY=y
+CONFIG_BT_MESH_FRIEND=y
+CONFIG_BT_MESH_ADV_BUF_COUNT=13
+CONFIG_BT_MESH_TX_SEG_MAX=10
+CONFIG_BT_MESH_PB_GATT=y
+CONFIG_BT_MESH_GATT_PROXY=y
+
+# Bluetooth Mesh models
+CONFIG_BT_MESH_ONOFF_SRV=y
+CONFIG_BT_MESH_LIGHTNESS_SRV=y
+
+CONFIG_BT_MESH_LIGHT_CTRL_SRV=y
+CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_FADE_ON=1000
+CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_FADE_PROLONG=1000
+CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_FADE_STANDBY_AUTO=1000
+CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_FADE_STANDBY_MANUAL=0
+CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_ON=3
+CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_PROLONG=3
+CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_ON=65535
+CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG=10000
+CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_STANDBY=0
```

Comment:
I see your point. I have some minor arguments:

- The prolong lvl value is set to this value since the visible difference between ON and PROLONG is a little hard to differentiate with the default lvl values.
- The default timeout for each state is a little long, so the user would need to wait for 1-2 minutes to see all the sample features from a single button push.
- The places where i have set the same value as the default was set there to make it easier for the user to change it without having to find the Kconfig settings them self (But i agree that this might be overkill).

Reply:
Done. Came to an agreement in person.